worlds/subnautica/__init__.py

```diff
@@ -65,22 +65,40 @@ def generate_early(self) -> None:
             creature_pool, self.options.creature_scans.value)
 
     def create_regions(self):
-        self.multiworld.regions += [
-            self.create_region("Menu", None, ["Lifepod 5"]),
-            self.create_region("Planet 4546B",
-                               locations.events +
-                               [location["name"] for location in locations.location_table.values()] +
-                               [creature + creatures.suffix for creature in self.creatures_to_scan])
-        ]
-
-        # Link regions
-        self.multiworld.get_entrance("Lifepod 5", self.player).connect(self.multiworld.get_region("Planet 4546B", self.player))
+        # Create Regions
+        menu_region = Region("Menu", self.player, self.multiworld)
+        planet_region = Region("Planet 4546B", self.player, self.multiworld)
+
+        # Link regions together
+        lifepod_5_connection = Entrance(self.player, "Lifepod 5", menu_region)
```

Comment:
same suggestion as i made for soe. this does the same thing as your 3 lines here under the hood, so this is absolutely being picky, it just reads a bit better imo.
```suggestion
        menu_region.connect(planet_region, "Lifepod 5")
```